src/modules/85-cv/hooks/useErrorBudgetRestHook/useErrorBudgetRestHook.utils.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+export const calculateErrorBudgetByIncrement = (currentErrorBudget: number, increment = NaN): number => {
```

Comment:
Both function are similar 

can't we can use calculateRemainingErrorBudgetByIncrement for both use cases ?

Reply:
keep remainingErrorBudget as optional as we can achieve using same funtion

Reply:
If we combine both, then we have to check whether it should return the total error budget or the remaining error budget. It's better if a function does one thing at a time. So it's better to keep them separate.

Reply:
Please consider if this can be refactored further  

return Math.round((remainingErrorBudget || currentErrorBudget) + currentErrorBudget * (increment / 100))
```suggestion
export const calculateErrorBudgetByIncrement = (currentErrorBudget: number, increment = NaN): number => {
```